Tensile/KernelWriter.py

```diff
@@ -3268,18 +3268,23 @@ def getSourceFileString(self, kernel):
 
     try:
       if kernel["KernelLanguage"] == "Assembly":
-        self.writeByteArrayScript()
-
         asmPath = self.getAssemblyDirectory()
-        coFile = self.getSingleCodeObjectFile(kernel)
         kernelName = self.getKernelName(kernel)
 
-        if globalParameters["CodeFromFiles"] or globalParameters["NewClient"] > 1:
-          # I guess in this case we are making sure that the code object file exists by executing the code
-          # above but we aren't placing it into the source.
+        if globalParameters["GenerateSourcesAndExit"]:
```

Comment:
A new global parameter should get a default value by assigning it in [Common.py](https://github.com/ROCmSoftwarePlatform/Tensile/blob/develop/Tensile/Common.py#L53).

Reply:
Will do.

Reply:
Done.